checkov/common/bridgecrew/run_metadata/abstract_run_metadata_extractor.py

```diff
@@ -0,0 +1,83 @@
+from abc import abstractmethod
+from checkov.common.bridgecrew.run_metadata.registry import registry
+from checkov.common.bridgecrew.run_metadata.ci_variables import (
+    BC_TO_BRANCH,
+    BC_PR_ID,
+    BC_PR_URL,
+    BC_COMMIT_HASH,
+    BC_COMMIT_URL,
+    BC_AUTHOR_NAME,
+    BC_AUTHOR_URL,
+    BC_RUN_ID,
+    BC_RUN_URL,
+    BC_REPOSITORY_URL,
+    BC_FROM_BRANCH,
+)
+
+
+class RunMetaDataExtractor:
+    from_branch = ""
+    to_branch = ""
+    pr_id = ""
+    pr_url = ""
+    commit_hash = ""
+    commit_url = ""
+    author_name = ""
+    author_url = ""
+    run_id = ""
+    run_url = ""
+    repository_url = ""
+
```

Comment:
please remove this, class level attributes are not needed here, and you don't leverage them 😄